lib/fastlane/plugin/wpmreleasetoolkit/actions/common/circleci_trigger_job_action.rb

```diff
@@ -0,0 +1,59 @@
+module Fastlane
+    module Actions    
+        class CircleciTriggerJobAction < Action
+            def self.run(params)
+                require_relative '../../helper/ci_helper.rb'
+                
+                UI.message "Triggering job #{params[:job_params]} on branch #{params[:branch]}"
+
+                ci_helper = Fastlane::Helper::CircleCIHelper.new(
+                    params[:circle_ci_token],
+                    params[:repository],
+                    params[:organization]
+                )
+
+                res = ci_helper.trig_job(params[:branch], params[:job_params])
+                (res.code == "201") ? UI.message("Done!") : UI.user_error!("Failed to start job\nError: [#{res.code}] #{res.message}")
+            end
+  
+            #####################################################
+            # @!group Documentation
+            #####################################################
+  
+            def self.description
+                "Triggers a job on CircleCI"
+            end
+  
+            def self.available_options
+            [
+                FastlaneCore::ConfigItem.new(key: :circle_ci_token,
+                                            env_name: "FL_CCI_TOKEN",
+                                            description: "CircleCI token",
+                                            is_string: true), 
+                FastlaneCore::ConfigItem.new(key: :organization,
+                                            env_name: "FL_CCI_ORGANIZATION",
+                                            description: "The GitHub organization which hosts the repository you want to work on",
+                                            is_string: true,
+                                            default_value: "wordpress-mobile"), 
+                FastlaneCore::ConfigItem.new(key: :repository,
+                                            env_name: "FL_CCI_REPOSITORY",
+                                            description: "The GitHub repository you want to work on",
+                                            is_string: true),  
+                FastlaneCore::ConfigItem.new(key: :branch,
+                                            env_name: "FL_CCI_BRANCH",
+                                            description: "The branch on which you want to work on",
+                                            is_string: true),
+                FastlaneCore::ConfigItem.new(key: :job_params,
+                                            env_name: "FL_CCI_JOB_PARAMS",
+                                            description: "Parameters to send to the CircleCI pipeline",
+                                            is_string: false,
```

Comment:
`is_string` is deprecated. You should use `type:` parameter instead. For example replace `is_string: true` with `type: String`; Replace `is_string: false` with the appropriate `type: <expected ruby type here>`.

The only use case left for `is_string: false` instead of `type: X` is when the expected type is a Boolean (because Ruby doesn't have formal `Boolean` _type name_ – only `TrueClass` and `FalseClass` and boolean literals)

For more information, see https://docs.fastlane.tools/advanced/actions/#using-fastlanecoreconfiguration